src/types/page.ts

```diff
@@ -0,0 +1,8 @@
+export interface PageOptions {
+    page?: number;
+}
+  
+export interface LocaleOptions extends PageOptions {
+    region?: string;
+    language?: string;
+}
```

Comment:
Using this would interfere with my newly created way to handle options by using union types on different base options interfaces to create the needed option parameters in my PR #12. I think we should decide on one way to handle Option parameters and maybe add one utility Class/Function which would handle everything like checking for undefined options.

Reply:
I'm kinda new to the whole web development world so forgive my lack of knowledge, but how does this interfere with your changes more than the naming on things? I can't find where you're using union types in your PR 🤔 

Anyway, I don't have any stong opinions regarding this. You (or maybe @blakejoy) can decide and I'll adapt

Reply:
@Der-Penz Have you given it any tought?

Reply:
Sorry for the delayed answer, I overlooked your comment. "Interfer" might be the wrong word. I just mean it would be a different way on how to at those options. I proposed to make an interface for each option and in the endpoint you could just add all of the options by using union types. you extended the PageOption interface, while I proposed to make all different interfaces and later add them with union types.
But as you already said @blakejoy should decide.

Reply:
You've been more intertwined with this as of lately so however you think will implement the necessary changes and not break anything. I'm for @Der-Penz @Hoidi

Reply:
Maybe you could merge my pull request #12  if everything is fine with it and after it, Hoidi can remove its PageOptions and Localoptions and update the type/options.ts to include a Localoptions interface which he can use then.

Reply:
Merged